Mathlib/Data/TypeVec.lean

```diff
@@ -669,7 +752,7 @@ theorem subtypeVal_toSubtype {α : TypeVec n} (p : α ⟹ «repeat» n Prop) :
 theorem toSubtype_of_subtype_assoc
     {α β : TypeVec n} (p : α ⟹ «repeat» n Prop) (f : β ⟹ Subtype_ p) :
     @toSubtype n _ p ⊚ ofSubtype _ ⊚ f = f := by
-  rw [← comp_assoc, toSubtype_of_subtype]; simp
+  rw [← comp_assoc, toSubtype_of_subtype]; rfl
```

Comment:
idem ditto here, if you extract the previous cleanup in a standalone PR, include this cleanup, too!

In general, have a look through the diff, and extract all such cleanups. If there's just a few, you could create a single "chore: clean TypeVec proofs" PR, but if there's a lot, it'd be better to even split them up in a few clean-up PRs. Especially as a new contributor, it's super helpful to start with very small, obvious PRs to get used to the process (and build trust with reviewers).